Packs/FortiSIEM/IncidentFields/incidentfield-FortiSIEM_Destination_User.json

```diff
@@ -0,0 +1,29 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "FortiSIEM"
+    ],
+    "caseInsensitive": true,
+    "cliName": "destinationuser",
```

Comment:
Please add a prefix to cliname.
should be `fortisiemdestinationuser`

Reply:
Done.